diesel_derives/src/util.rs

```diff
@@ -8,6 +8,13 @@ pub fn wrap_in_dummy_mod(item: TokenStream) -> TokenStream {
     quote! {
         #[allow(unused_imports)]
         const _: () = {
+            // This import is not actually redundant. When using diesel_derives
+            // inside of diesel, `diesel` doesn't exist as an extern crate, and
+            // to work around that it contains a private
+            // `mod diesel { pub use super::*; }` that this import will then
+            // refer to. In all other cases, this imports refers to the extern
+            // crate diesel.
+            #[allow(clippy::single_component_path_imports, clippy::unknown_clippy_lints)]
```

Comment:
I think this will be fixed next clippy release: rust-lang/rust-clippy#5183 (but we can remove this later.)

Reply:
Ah, nice. I don't particularly care whether it is silenced explicitly here if no future beta or stable clippy emits it from inside macros.

Should I remove the lint attribute?

Reply:
Given @weiznich's change suggestion, I've removed it.